### PR TITLE
Update CruiseInformation.csv - Papa 7a cruise

### PR DESCRIPTION
**Pull request by cdobs**
Papa 7a cruise quick-look report 3203-00705

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -280,4 +280,5 @@ EK20200814,R/V Elakha,2020-08-14T14:15:00,2020-08-14T18:50:00,Endurance CSPP dep
 TS20200819,F/V Tenacious II,2020-08-19T07:30:00,2020-08-20T06:30:00,Pioneer 14b Glider deployments and recoveries
 PS20200917,R/V Pacific Storm,2020-09-17T17:30:00,2020-09-20T17:30:00,Endurance CSPP and glider operations
 TS20200925,F/V Tenacious II,2020-09-25T21:00:00,2020-09-26T15:00:00,Pioneer 14c Glider deployments
+ZP20200930,R/V Zephyr,2020-09-30T17:00:00,2020-10-03T23:59:59,Papa 7a Glider deployments
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:283]: looks good; Zephyr is already in vessel list.

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1354,6 +1354,14 @@ GP05MOAS-PG515-02-DOSTAM000,Global Station Papa,Mobile Assets,Profiling Glider 5
...
 GP05MOAS-PG515-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 515,Nitrate,Satlantic,SUNA V2,0,200
+GP05MOAS-PG566,Global Station Papa,Mobile Assets,Profiling Glider 566,,,,0,200
+GP05MOAS-PG566-00-ENG000000,Global Station Papa,Mobile Assets,Profiling Glider 566,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+GP05MOAS-PG566-01-CTDGVM000,Global Station Papa,Mobile Assets,Profiling Glider 566,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+GP05MOAS-PG566-02-DOSTAM000,Global Station Papa,Mobile Assets,Profiling Glider 566,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+GP05MOAS-PG566-03-FLORTM000,Global Station Papa,Mobile Assets,Profiling Glider 566,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+GP05MOAS-PG566-04-FLORTO000,Global Station Papa,Mobile Assets,Profiling Glider 566,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
+GP05MOAS-PG566-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 566,Nitrate,Satlantic,SUNA V2,0,200
+GP05MOAS-PG566-06-PARADM000,Global Station Papa,Mobile Assets,Profiling Glider 566,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:1364]: looks good

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:283]: looks good; confirmed Zephyr already on vessel list

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -1220,6 +1220,7 @@ CGINS-DOSTAM-00480,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA
 CGINS-DOSTAM-00494,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,494,,,,CGSN Glider Instrument Pool,
 CGINS-DOSTAM-00496,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,496,,,,CGSN Glider Instrument Pool,
 CGINS-DOSTAM-00499,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,,Aanderaa,Optode 4831,499,,,,CGSN Glider Instrument Pool,
+CGINS-DOSTAM-00537,,Sensor,1,Dissolved Oxygen Stable Response for Glider: DOSTA Series M,DOSTA for Slocum Glider,Aanderaa,Optode 4831,537,,,,CGSN Glider Instrument Pool,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:1223]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:1223]: Looks good.

File: vocab/vocab.csv (hunk 2 above)
Comment (snwhite) [reply vocab/vocab.csv:1364]: Looks good.

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv:283]: Looks good.  Do we expect to have an actual Zephyr return time?

**Review by snwhite — APPROVED** [review]

**Pull request merged**